src/test/resources/schema/oracle/oracle-migration-v4.sql

```diff
@@ -1,4 +1,3 @@
-ALTER TABLE "journal" MODIFY "message" BLOB DEFAULT NULL;
```

Comment:
Should this be deleted? Why?
(same for the snapshot column in the snapshot table)

Reply:
it does not work:
```
sql> ALTER TABLE "journal" MODIFY "message" BLOB DEFAULT NULL [2018-09-18 23:08:08] [99999][22859] ORA-22859: invalid modification of columns
```